api/report_server.thrift

```diff
@@ -43,16 +53,17 @@ struct ReviewData{
 
 //-----------------------------------------------------------------------------
 struct ReportData{
-  1: string               checkerId,       // the qualified id of the checker that reported this
-  2: string               bugHash,         // This is unique id of the concrete report.
-  3: string               checkedFile,     // this is a filepath
-  4: string               checkerMsg,      // description of the bug report
-  5: i64                  reportId,        // id of the report in the current run in the db
-  6: i64                  fileId,          // unique id of the file the report refers to
-  7: shared.BugPathEvent  lastBugPosition  // This contains the range and message of the last item in the symbolic
-                                           // execution step list.
-  8: shared.Severity      severity         // checker severity
-  9: ReviewData           review           // bug review status informations.
+  1: string              checkerId,       // the qualified id of the checker that reported this
+  2: string              bugHash,         // This is unique id of the concrete report.
+  3: string              checkedFile,     // this is a filepath
+  4: string              checkerMsg,      // description of the bug report
+  5: i64                 reportId,        // id of the report in the current run in the db
+  6: i64                 fileId,          // unique id of the file the report refers to
+  7: shared.BugPathEvent lastBugPosition  // This contains the range and message of the last item in the symbolic
+                                          // execution step list.
+  8: shared.Severity     severity         // checker severity
+  9: ReviewData          review           // bug review status informations.
+  10: DetectionStatus    detectionStatus  // 'new', 'resolved', 'unresolved', 'reopened'
```

Comment:
I don't think we should put a comment here, the enum explains the possible values.